pkg/generator/ignition/generator.go

```diff
@@ -181,3 +177,89 @@ func getGardenerDecoder() runtime.Decoder {
 	}
 	return decoder
 }
+
+func additionalContainterdConfigFile(mirrors []metalextensionv1alpha1.RegistryMirror) types.File {
+	containerdConfig := types.File{
+		Path:       "/etc/containerd/config.toml",
+		Filesystem: "root",
+		Mode:       &types.DefaultFileMode,
+		Contents: types.FileContents{
+			Inline: containerdBaselineConfig,
+		},
+	}
+	if len(mirrors) == 0 {
+		return containerdConfig
+	}
+
+	renderedContent := `# Generated by os-extension-metal
+imports = ["/etc/containerd/conf.d/*.toml"]
+version = 2
+
+[plugins."io.containerd.grpc.v1.cri".registry]
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
+`
+	for _, m := range mirrors {
+		for _, of := range m.MirrorOf {
+			renderedContent += fmt.Sprintf(`    [plugins."io.containerd.grpc.v1.cri".registry.mirrors.%q]
```

Comment:
Missing newline ?

Reply:
The newlines are on line 200 and 205 or am I missing something?